pipelines/resources/tekton/task/func-deploy/0.1/func-deploy.yaml

```diff
@@ -16,17 +16,15 @@ spec:
   - name: path
     description: Path to the function project
     default: ""
+  - name: image
+    description: Container image to be deployed
+    default: ""
   workspaces:
     - name: source
       description: The workspace containing the function project
   steps:
   - name: func-deploy
     image: "ghcr.io/knative-sandbox/kn-plugin-func/func:latest"
-    command: ["func"]
-    args:
-      - "deploy"
-      - "--verbose"
-      - "--build=disabled"
-      - "--push=false"
-      - "--path"
-      - "$(params.path)"
+    script: |
+      [[ "$(params.image)" != "" ]] && export FUNC_IMAGE="$(params.image)"
```

Comment:
Minor nit, but you don't really need the initial check for `""` since setting `FUNC_IMAGE=""` is essentially a noop anyway.

Reply:
Ah that's right! My first iteration on this was setting the image via parameter, then changed it to env variable and haven't thought it properly. Thanks!